src/signTxOutput.c

```diff
@@ -120,6 +135,13 @@ static void signTx_handleOutput_address_ui_runStep()
 
 	UI_STEP_BEGIN(subctx->ui_step, this_fn);
 
+	UI_STEP(HANDLE_OUTPUT_ADDRESS_BYTES_STEP_WARNING_DATUM) {
+		ui_displayPaginatedText(
+		        "WARNING: output",
+		        "missing datum (could be required)",
```

Comment:
I'm not sure whether this wording is strong enough -- "could be required" sounds like "the tx could be refused by the blockchain if not provided" to me, whereas the reality is that the tx will be accepted and the UTXO will be unspendable (!).

Reply:
What about
```
		        "WARNING: output",
		        "could be unspendable due to missing datum",
```

Reply:
That sounds good